data_warehouse/services/aggregate_data_service.py

```diff
@@ -261,7 +269,7 @@ def get_group_coverage_data(cls, context):
         end_time = make_aware(
```

Comment:
Why are you still validate your parameters in these functions, by now you should do these in your validators.